pdr_backend/lake/test/test_etl.py

```diff
@@ -41,7 +41,7 @@ def test_etl_tables(
     assert len(pdr_payouts_df) == 4
     assert len(pdr_predictions_df) == 5
     assert len(pdr_truevals_df) == 5
-    assert len(TableRegistry().get_tables()) == 7
+    assert len(TableRegistry().get_tables()) == 4
```

Comment:
instead of hardcoding the tests, I think we should assert lengths like either:
- length of bronze table types
- length of etl table types
- combinations of the two

this will make it s.t. we don't have to modify tests when we add/remove tables from the defined list

Reply:
Yes, I've been trying to update some of these. I'll do it here too.

Reply:
I have fixed this one and several others...